content/docs/concepts/csidriver/features/powerstore.md

```diff
@@ -764,3 +764,28 @@ To configure how often driver checks for changed capacity set `storageCapacity.p
 The CSI PowerStore driver supports the provisioning of Metro volumes. The process and details of how to provision and use Metro volumes can be found [here](../../../replication/high-availability).
 
 Please note that the Metro feature does not require the deployment of the replicator sidecar or the replication controller.
+
+## Shared NFS
+Shared NFS utilizes native NFS features to support large-scale ReadWriteMany (RWX) volumes through the access mode, enabling efficient shared storage across multiple consumers. It follows a client-server model, with a node serving as an NFS server—and potentially a client as well.
+- **Scalability and Flexibility**: Offers enhanced scalability and flexibility compared to traditional NFS, which relies on a single dedicated server.
+- **Centralized File Management**: Allows remote files to be accessed as if they were local, simplifying file management and reducing duplication for more efficient storage use.
+- **NFSv4 Recommendation**: NFS versions v4,1, v4.2.
+- **Prerequisites**: NFS-related services (nfs-server and nfs-mountd on Linux) must be running on all participating worker nodes.
+- **CSI PowerStore Support**: Version 2.14 introduces support for Shared NFS via a new StorageClass.
```

Comment:
Does it need to state "CSI PowerStore Support" or can it just indicate that our first supported storage array is PowerStore?

Reply:
Updated wording to "Enable Shared NFS Support" since this is already on the PowerStore features page and doesn't need to mention "CSI PowerStore" again.